backend/src/app.module.ts

```diff
@@ -1,12 +1,30 @@
 import { Module } from '@nestjs/common';
+import { TypeOrmModule, TypeOrmModuleOptions } from '@nestjs/typeorm';
+import { config } from 'dotenv';
 
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { BooksModule } from './books/books.module';
+import { Book } from './books/book.entity';
+
+config();
+
+const { DB_PORT, DB_USERNAME, DB_PASSWORD, DB_DATABASE, DB_HOST } = process.env;
+
+const ormOptions: TypeOrmModuleOptions = {
```

Comment:
Suggestion: Let's move these to an own configuration file e.g. `configuration/configuration.ts`